config/initializers/logging.rb

```diff
@@ -0,0 +1,20 @@
+config = Rails.application.config
```

Comment:
Should this file be named something like `00_logging.rb`, so it's one of the first things loaded?

If any of the other initializers log something, it might go to the wrong place.